pkg/flock/flock_int_test.go

```diff
@@ -0,0 +1,32 @@
+package flock_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/rancher/k3s/pkg/flock"
+)
+
+var lock int
+var _ = Describe("file locks", func() {
+	When("a new exclusive lock is created", func() {
+		It("starts up with no problems", func() {
+			var err error
+			lock, err = flock.Acquire("/tmp/testlock.test")
```

Comment:
Can you make `/tmp/testlock.test` a const?

Reply:
Fixed.